### ISSUE TITLE
死亡後復活

### ISSUE THREAD

**Issue by lyhyl** [Project-F/F.LF#2]
![2016-03-26](https://cloud.githubusercontent.com/assets/3370369/14057935/63b8db9c-f34a-11e5-8a10-adee5a09a377.png)
在被冰凍後死亡會復活

**Comment by tyt2y3** [comment]
謝謝你的報告 :D 不過現在我可能未有時間fix
thanks for the reporting.

**Comment by DaKnig** [comment]
happened to me, I was the one who was standing after being dead. I could even move and walk around. and they said the walking dead will never come...

**Comment by olivertribe** [comment]
I can't seem to replicate this one. 
I've tried:
1. Freezing someone as a killing blow. (so they're frozen at time of death). The round ends, the ice breaks and they stay on the ground. 
2. Freezing someone multiple times, and a normal kill. Same as above. 
3. Freezing and breaking out of freeze with killing blow. same as above

Could you provide further instructions to replicate?

**Comment by mcwissink** [comment]
An easy way to reproduce this issue is to use attacks that self-harm. If you use Firen's explosion to kill himself, he will remain standing after death.

**Comment by DaKnig** [comment]
in the original game firen can't kill himself with explosion. the attack requires 40 health.

**Comment by tyt2y3** [comment]
Thank you everyone for the info. I wasn't aware of these facts!

**Comment by DaKnig** [comment]
I feel like this project works completely differently on the logic of the engine level compared to the original.

**Comment by tyt2y3** [comment]
Yes, this is one of the goals indeed. A clean-room engineered design, but not a logical clone. They should be compatible, but not the same thing.
At least when I started this project, I thought it would give more future to the longevity of the project.

**Comment by DaKnig** [comment]
clean room does not mean not cloning the logic. it only means you dont get any inside info. I got to understand the internal logic somewhat by playing the game ... a bit too much. sadly I dont think I could be of any use to this project besides reporting some bugs...

**Comment by tyt2y3** [comment]
> clean room does not mean not cloning the logic. it only means you dont get any inside info. I got to understand the internal logic somewhat by playing the game ... a bit too much. sadly I dont think I could be of any use to this project besides reporting some bugs...

It's definitely fine. This project is perhaps our tribute.

I will try to fix them when I got the leisure time.
